.travis/install_openssl.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+OPENSSL="OpenSSL_1_1_1c"
+OPENSSL_ZIP="${OPENSSL}.zip"
+travis_retry wget --no-check-certificate https://github.com/openssl/openssl/archive/${OPENSSL_ZIP}
```

Comment:
Can't this be solved in another way? Just disabling cert check doesn't sound a good idea.

Reply:
OK.